backend/Utils/Edna.Utils.Http/Edna.Utils.Http/EdnaExternalHttpHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Net.Security;
+
+namespace Edna.Utils.Http
+{
+    public class EdnaExternalHttpHandler: HttpClientHandler
+    {
+        public static string Name = nameof(EdnaExternalHttpHandler);
+        public EdnaExternalHttpHandler() : base()
```

Comment:
We may also consider a single static function `CreateDefault` to consolidate all the scattered `new` across the codebase. What this helps is w.r.t passing any additional params or config at the time of creating or performing additional settings if needed inside the `CreateDefault` function itself.

Reply:
Thanks for this comment, I have added modifications to address this! It has made the code a lot cleaner indeed.